src/theme.js

```diff
@@ -114,6 +121,24 @@ export const fontWeights = {
   bold: 700
 };
 
+export const icons = {
+  annotation: '\f039',
+  calendar: '\f035',
+  caret: '\f037',
+  chatBubbleOutline: '\f003',
+  checkboxChecked: '\f030',
+  checkboxIndeterminate: '\f02f',
+  checkboxMarkedCircleOutline: '\f000',
+  chevron: '\f03a',
+  crLogo: '\f02e',
+  download: '\f004',
+  swap: '\f038',
+  symbolCircle: '\f034',
+  thumbDownOutline: '\f001',
+  thumbUpOutline: '\f002',
+  trash: '\f036'
+};
```

Comment:
I went back and forth between making a constant for icons outside of the theme, but eventually landed here. The thought being you shouldn't need to reference the icon character directly. You should be able to use the `<Icon />` component in most cases. The select exposed a circumstance where you would want to use the icon without using the component.

Reply:
Agreed. At no point should anyone have to remember that \f003 is chatBubbleOutline.